io-engine/src/bdev/nvmx/mod.rs

```diff
@@ -19,7 +19,7 @@ use crate::{
 mod channel;
 mod controller;
 mod controller_inner;
-mod controller_state;
+pub mod controller_state;
```

Comment:
It's better to add an accessor to NvmeController to get the state rather than exposing the state.